lua/rust-tools/dap.lua

```diff
@@ -64,6 +60,16 @@ function M.get_codelldb_adapter(codelldb_path, liblldb_path)
         end)
       end
     end)
+
+    vim.defer_fn(function()
+      vim.schedule(function()
```

Comment:
`defer_fn` automatically wraps the function in `vim.schedule`, maybe
this `vim.schedule` can be removed